README.md

```diff
@@ -0,0 +1,24 @@
+# Flarum documentation
+
+Hi! Looking to contribute documentation to [Flarum](http://flarum.org)? That is great, we can use all the help. Flarum is evolving rapidly and so should its documentation.
+
+_This repo will eventually be deprecated in favour of [flarum/docs](https://github.com/flarum/docs). But until a definitive conversion date is planned, you are welcome to contribute here._
+
+# Get started 
+
+You can edit small changes online using the Github editor. For larger edits it is recommended to make a local checkout and test your changes before you commit them. To do that, you need Ruby. Instructions for Linux/Mac:
+
+```
+git clone https://github.com/flarum/flarum.github.io
+cd flarum.github.io
+
+cat > Gemfile <<EOM
+source 'https://rubygems.org'
+gem 'github-pages', group: :jekyll_plugins
+EOM
```

Comment:
Hmm, interesting, I simply installed Jekyll globally and did not need this plugin gem. What does it do?

Reply:
Not sure, but instructions taken from https://help.github.com/articles/setting-up-your-github-pages-site-locally-with-jekyll/